scripts/reduce_CG1D.py

```diff
@@ -79,7 +75,7 @@ def main(inputfile: Union[str, Path], outputdir: Union[str, Path]) -> int:
     # step 0: check if data is ready for reduction
     if not auto_reduction_ready(inputfile):
         logger.warning("Data incomplete, waiting for next try.")
-        return 1
+        return ERROR_GENERAL  # TODO need to discus this
```

Comment:
maybe we could have one `SCAN_INCOMPLETE`  exit code. After all, this event is not an error